examples/mxnet/gcmc/train.py

```diff
@@ -0,0 +1,269 @@
+import os, time
+import argparse
+import logging
+import random
+import string
+import numpy as np
+import mxnet as mx
+from mxnet import gluon
+from data import MovieLens
+from model import GCMCLayer, BiDecoder, InnerProductLayer
+from utils import get_activation, parse_ctx, gluon_net_info, gluon_total_param_num, params_clip_global_norm, \
+    logging_config, MetricLogger
+from mxnet.gluon import nn, HybridBlock, Block
```

Comment:
`logging_config`, `nn`, `HybridBlock` are not used.

Reply:
Done